api-errors.json

```diff
@@ -203,6 +203,10 @@
         "short_message": "Invalid Barcode",
         "long_message": "The barcode could not be created due to invalid barcode type"
     },
+    "40099": {
```

Comment:
I'm not sure why there is a `40098` error, but this one should probably go after [40054](https://github.com/kivra/kivra-api-errors/blob/main/api-errors.json#L198)